app/jinja_filters.py

```diff
@@ -145,9 +145,12 @@ def calculate_years_difference(from_str, to_str):
 @evalcontextfunction
 def format_date_range(context, start_date, end_date=None):
     if end_date:
-        return '{from_date} to {to_date}'.format(from_date=format_date(context, start_date),
-                                                 to_date=format_date(context, end_date))
-    return format_date(context, start_date)
+        result = '{from_date} to {to_date}'.format(from_date=format_date(context, start_date),
+                                                   to_date=format_date(context, end_date))
+    else:
+        result = format_date(context, start_date)
```

Comment:
format date ends in a call to mark safe . I can see it both ways . At least its explicit here under all circumstances and it will be a trivial cost to leaving it this way - explicit over implicit and all that ....

Reply:
Good point. Let's just pretend that I intended to do it this way and it wasn't just a happy oversight :)